méli-mélo/2024-04-stepsquiz/js/stepsquiz.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @title WET-BOEW Steps Quiz plugin
+ * @overview Quiz pattern for use with the Steps Form plugin.
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author ricokola
+ */
+( function( $, document ) {
+"use strict";
+
+//Detect the enhancement of the quiz
+var instances = document.querySelectorAll( ".wb-steps.quiz" );
+
+//How many quiz instances in the page
+instances.forEach ( ( instance ) => {
+	let $instance = $( instance );
+
+	// Calculate number of questions
+	let numQuestion = $( "fieldset", instance ).length;
```

Comment:
Optional: Can you try to use the variable `$instance` instead of `instance`? Technically, it should work.